code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -960,9 +959,16 @@
 		else
 			actions.start(new/datum/action/bar/flock_repair(F), user)
 	else
-		..()
+		src.attack_hand(target, user)
 
+//why doesn't attack_hand trigger on mobs aaa
 /datum/limb/flock_converter/disarm(atom/target, var/mob/living/critter/flock/drone/user)
+	src.attack_hand(target, user)
+
+/datum/limb/flock_converter/grab(atom/target, var/mob/living/critter/flock/drone/user)
+	src.attack_hand(target, user)
```

Comment:
What about just checking for the target is a mob if needed, and doing try_cage on that, instead of calling attack_hand? Don't think the intent override would be needed in that case?

Reply:
This proc isn't even called without the intent override, it's stopped in the critter parent.

Reply:
Ok I see. I guess either works, but still feels bit confusing though, since I think it will be meaning that attack_hand handles some mob attacking stuff while the other intents handle all other mob attacking? Feel it would be more clear if attack_hand handled only non-mob attacking like originally, but again I guess either works, idk

Reply:
🤷‍♀️

Reply:
Idk, still do kind of think using try_cage instead of attack_hand, can maybe also get Amy's input?